Frontend/student-hub/src/pages/Profile.js

```diff
@@ -319,15 +340,24 @@ function Profile() {
             />
           </Grid>
           <Grid item xs={1}>
-            <TextField
-              fullWidth
-              type="text"
-              label="School"
-              variant="outlined"
+          <FormControl fullWidth>
+            <InputLabel>School</InputLabel>
+            <Select
               name="school"
-              value={profile.school || ''}
+              value={profile.school}
```

Comment:
Can `school` be null? If so, what would the UI look like with a null school? Can users remove the school if they want to go back to not having a school?